projekt/src/tekstovniVmesnik/tekstovniVmesnik.ml

```diff
@@ -35,16 +35,19 @@ let update model = function
             stanje_vmesnika = RezultatPrebranegaNiza;
           })
   | ZamenjajVmesnik stanje_vmesnika -> { model with stanje_vmesnika }
+  | VrniVPrvotnoStanje -> { model with stanje_avtomata = zacetno_stanje model.avtomat}
```

Comment:
V kakšnem stanju je vmesnik? Verjetno je trenutno edina možnost, da je bil pred tem v `SeznamMoznosti`, ampak je bolje, če to nastavimo eksplicitno.